libr/bin/p/bin_elf.c

```diff
@@ -881,11 +881,19 @@ static void _patch_reloc (ut16 e_machine, RIOBind *iob, RBinElfReloc *rel, ut64
 }
 
 static bool ht_insert_intu64(SdbHash* ht, int key, ut64 value) {
-	return ht_insert (ht, sdb_fmt (-1, "%d", key), (void *)value);
+	ut64 *mvalue = malloc(sizeof(ut64));
```

Comment:
missing space before each (

Reply:
also null deref in here

Reply:
also, why not use dict, and avoid converting numbers to strings and allocating them in heap?

Reply:
i'm not using dict because I think it's a bad implementation of an hash table :) so I prefer to keep using ht.